api/src/Page/Sample.php

```diff
@@ -168,7 +168,7 @@ class Sample extends Page
         array('/sub/:ssid', 'put', '_update_sub_sample_full'),
         array('/sub', 'post', '_add_sub_sample'),
         array('/sub/:ssid', 'delete', '_delete_sub_sample'),
-        array('/sub/queue/cid/:cid', 'post', '_queue_all_sub_samples'),
+        array('/sub/queue/cid/:cid', 'get', '_queue_all_sub_samples'),
```

Comment:
Even if no data is being sent, I think it'd be better for this to be a POST request, since it queues all sub samples, rather than being a read-only operation

Reply:
Agree.